### PR TITLE
Dodane EmployeeDto.java oraz DepartmentDto.java,; 

### PR DESCRIPTION
**Pull request by MaciejKo**
…tak: Date date = new SimpleDateFormat("dd/MM/yyyy").parse("12/05/2015");

### CODE REVIEW

**Review by KamilaoPL — APPROVED** [review]

File: src/dto/EmployeeDto.java [hunk 1]
```diff
@@ -1,4 +1,54 @@
...
+
+    public String getOccupation() {
+        return occupation;
+    }
+
+    public void setOccupation(String occupation) {
+        this.occupation = occupation;
+    }
+
+    public Date getDane() {
```
Comment (RafalRomanQW) [inline src/dto/EmployeeDto.java]: getDane() ? i dont understand what it does

File: src/dto/EmployeeDto.java [hunk 2]
```diff
@@ -1,4 +1,54 @@
 package dto;
 
+import java.util.Date;
+
 public class EmployeeDto {
+    private String name;
+    private String surname;
+    private String occupation;
+    private Date date;
+    public DepartmentDto getDepartment() {
```
Comment (RafalRomanQW) [inline src/dto/EmployeeDto.java]: First all variables then getters and setters

File: src/dto/EmployeeDto.java [hunk 3]
```diff
@@ -1,4 +1,54 @@
...
+        return department;
+    }
+
+    public void setDepartment(DepartmentDto department) {
+        this.department = department;
+    }
+
+    private DepartmentDto department;
+
+    private int numberOfExperience;
```
Comment (RafalRomanQW) [inline src/dto/EmployeeDto.java]: what is the numberOfExperience ? Why do you need it ? I did not saw any requirement for this

**Review by RafalRomanQW — CHANGES_REQUESTED** [review]
Please fix my comments or explain why do we need this

**Review by RafalRomanQW — APPROVED** [review]

**Review by Karol-Plocharski — APPROVED** [review]

**Pull request merged**